webapp/e2e/package.json

```diff
@@ -33,7 +36,9 @@
     "path": "0.12.7",
     "pg": "8.3.3",
     "recursive-readdir": "2.2.2",
+    "reportportal-client": "5.5.0",
     "shelljs": "0.8.4",
+    "start-server-and-test": "1.11.3",
     "uuid": "8.3.0",
```

Comment:
Can you elaborate a bit why these new dependencies are needed?

Reply:
They are likely not needed. I just copied over the dependencies used in the webapp repo's e2e folder. I'll remove the unnecessary ones